app/src/main/java/com/depromeet/team6/data/background/FcmService.kt

```
@@ -30,9 +32,17 @@ class FcmService : FirebaseMessagingService() {
     @Inject lateinit var homeRepository: HomeRepository
 
     @Inject lateinit var authRemoteDataSource: AuthRemoteDataSource
+
+    private val serviceScope = CoroutineScope(Dispatchers.IO + SupervisorJob())
+
     private lateinit var body: String
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)